classes.py

```diff
@@ -0,0 +1,18 @@
+
+import subprocess
+
+class SolarCarProcess:
+        
+    def __init__(self, path):
+        self.path = path
+        self.process = subprocess.Popen(path)
+        self.timesRestarted = 0
+        
+    def start(self):
```

Comment:
SolarCarProcess can probably keep track of it's own timesRestarted, since it knows everytime it's `start()` function is called.